### PR TITLE
Add general graduate scholarship deposit item

### PR DESCRIPTION
**Pull request by rawOrlando**
Adds a new despot type general graduate scholarship

- [ ] Will need to manually go to add a new deposit type on dev and prod after running
- [x] This does not currently add to the correct collection. 
           The collection (nk322d32h) does not have a call number which is currently how the code.
           Not every collection should have a call number
           Sari is looking into if this collection should have one

https://tuftswork.atlassian.net/jira/software/c/projects/TDLR/boards/58?selectedIssue=TDLR-2492

### CODE REVIEW

File: config/deposit_type_seed.csv [hunk 1]
```diff
@@ -1,7 +1,7 @@
 display_name,deposit_view,license_name,deposit_agreement
 Faculty Scholarship,faculty_scholarship,Tufts Non-Exclusive Deposit License v1.0,"I understand and acknowledge that this submission is governed by the <a href=""/contribute/license"" rel=""nofollow"">Tufts Non-Exclusive Deposit License</a>."
 Fletcher School Capstone Project,capstone_project,Tufts Non-Exclusive Deposit License: Fletcher v1.0,"I understand and acknowledge that this submission is governed by the <a href=""/contribute/license"" rel=""nofollow"">Tufts Non-Exclusive Deposit License</a>."
-GIS Expo Student Winners,gis_poster,Tufts Non-Exclusive Deposit License v1.0,"I understand and acknowledge that this submission is governed by the <a href=""/contribute/license"" rel=""nofollow"">Tufts Non-Exclusive Deposit License</a>.
+GIS Expo Student Winners,gis_poster,Tufts Non-Exclusive Deposit License v1.0,"I understand and acknowledge that this submission is governed by the <a href=""/contribute/license"" rel=""nofollow"">Tufts Non-Exclusive Deposit License</a>.
```
Comment (rawOrlando) [inline config/deposit_type_seed.csv:4]: I don't see how theses 2 lines are different, I did not mean to change anything here.

**Comment by rawOrlando** [comment]
The code needs to find something else to look up on other than call number.

**Comment by rawOrlando** [comment]
This implementation gets rid of the when these collections get deleted, they would get rebuilt. In this case, the db would need to be reset so that the IDs can be freed up.  I don't see this case happening and do not think it is imporant

File: app/models/forms/grad_scholarship.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    super
+    @tufts_pdf.subject = [department]
+    @tufts_pdf.creator_department = [creator_dept]
+  end
+
+  private
+
+  # TODO: copied from Honors Thesis, problally needs work in both places
+  def creator_dept
+    terms = Qa::Authorities::Local.subauthority_for('departments').all
```
Comment (mkorcy) [inline app/models/forms/grad_scholarship.rb:25]: probably an aside but we haven't updated this department list in a long time, maybe as a follow up after this is done we should see if there needs to be any udpates.

**Review by mkorcy — APPROVED** [review]
see note on follow up ticket

**Pull request merged**